index.html

```diff
@@ -3857,6 +3857,72 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
```

Comment:
IMO, this section is confusing to readers and implementers. I don't think it adds a lot to list steps where JSON-processing is used.

Reply:
Are you saying the list of examples is confusing? The reason it was added was because people wanted to know about scenarios where implementers didn't do JSON-LD processing as highlighted in https://github.com/w3c/vc-data-model/pull/1088#issuecomment-1591275654 

The list are specific examples of when implementers don't do JSON-LD processing today (and just process as JSON). 

What concrete suggestions do you have to improve this text given the above?

Reply:
I keep struggling with this a bit.  As an author of some extensive crawling / search and other functions that work with JSON-LD a lot we have never felt the need to process JSON-LD as other than JSON except in cases where we needed for some reason to operate on that data as RDF.  Could we rephrase this as something like:

```
While the base media type for data types described in this specification
is JSON-LD, JSON-LD processing is not required to be performed, 
since JSON-LD is JSON.  Some scenarios where processing a 
verifiable credential as JSON only is desirable include, but are not limited to:
```

Then, perhaps we could add a corresponding section that talks about when you would want to leverage the graph properties of JSON-LD to get things into RDF.

Reply:
This has been implemented in https://github.com/w3c/vc-data-model/commit/d4e3d74312e6f480958111ee8f03d6fbd860cf0a .

> perhaps we could add a corresponding section that talks about when you would want to leverage the graph properties of JSON-LD to get things into RDF.

I suggest we handle this in another PR. Please feel free to open a new issue if you'd like a section on that.